pkg/configure/oneagent/pmc/ruxit/types_test.go

```diff
@@ -0,0 +1,192 @@
ection":"agentType","key":"java","value":"on"},{"section":"general","key":"addContainerNameToPGI","value":"off"},{"section":"general","key":"expandJavaAtFiles","value":"off"},{"section":"general","key":"disableAgentTypeBasedInjection","value":"off"},{"section":"general","key":"addNodejsScriptNameToPGI","value":"on"},{"section":"general","key":"enableNodeJsAgentEsmLoaders","value":"off"},{"section":"general","key":"cassandraClusterNameInPG","value":"on"},{"section":"general","key":"disableSpringBootGroupCalc","value":"off"},{"section":"general","key":"enablePodmanInjection","value":"off"},{"section":"general","key":"enableNodeJsAgentPreloading","value":"on"},{"section":"general","key":"tenant","value":"zib50933"},{"section":"general","key":"enableTibcoBWContainerEditionGroupCalc","value":"on"},{"section":"general","key":"switchToPhpAgentNG","value":"on"},{"section":"general","key":"fullStackJavaInMBProcesses","value":"on"},{"section":"general","key":"envoyInjection","value":"on"},{"section":"general","key":"containerInjectionRules","value":"-1::EXCLUDE:EQUALS,KUBERNETES_CONTAINERNAME,POD;-2::EXCLUDE:CONTAINS,IMAGE_NAME,pause-amd64;-3::EXCLUDE:EQUALS,KUBERNETES_NAMESPACE,openshift-sdn;-4::EXCLUDE:ENDS,KUBERNETES_FULLPODNAME,-build"},{"section":"agentType","key":"sdk","value":"on"},{"section":"general","key":"containerdInjection","value":"on"},{"section":"agentType","key":"apache","value":"on"},{"section":"agentType","key":"nodejs","value":"on"},{"section":"general","key":"dockerWindowsInjection","value":"off"},{"section":"agentType","key":"varnish","value":"off"},{"section":"agentType","key":"python","value":"off"},{"section":"general","key":"dockerInjection","value":"on"},{"section":"agentType","key":"plugin","value":"on"},{"section":"general","key":"runcInjection","value":"on"},{"section":"general","key":"trustedTimestampVerificationJavascriptAgent","value":"off"},{"section":"general","key":"enableWebSphereLibertyGroupCalc","value":"off"},{"section":"general","key":"optimizedSuspendThreads","value":"on"},{"section":"agentType","key":"dotnet","value":"not-global"},{"section":"general","key":"php80InjectionEA","value":"on"},{"section":"general","key":"stripIdsFromKubernetesNamespace","value":"on"},{"section":"general","key":"pythonInjection","value":"off"},{"section":"agentType","key":"iis","value":"on"},{"section":"general","key":"coreclrInjection","value":"on"},{"section":"general","key":"crioInjection","value":"on"},{"section":"general","key":"enableEquinoxGroupCalc","value":"on"},{"section":"agentType","key":"loganalytics","value":"on"},{"section":"agentType","key":"network","value":"on"},{"section":"general","key":"enableNodeJsMultiversionLibrary","value":"off"},{"section":"agentType","key":"go","value":"not-global"},{"section":"general","key":"enableOsAgentDefaultIdCalc","value":"off"},{"section":"general","key":"php81Injection","value":"on"},{"section":"general","key":"stripVersionFromImageName","value":"on"}],"revision":3551371520520734114}
```

Comment:
why to to use `tesdata/` similar to https://github.com/Dynatrace/dynatrace-operator/tree/main/pkg/otelcgen/testdata, it's easy to see diffs.